mod/wet4/views/default/input/groups_autocomplete.php

```diff
@@ -52,19 +57,19 @@
 					source: function( request, response ) {
 						$.getJSON( "<?php echo $site_url; ?>groups_autocomplete", {
 							q: request.term,
-							'groups_guids': function() {
-								var result = "";
+							 'groups_guids': function() {
```

Comment:
Looks like the rest of the changes below are an accidental whitespace change that added a space in front of the tabs